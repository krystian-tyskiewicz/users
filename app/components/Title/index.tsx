'use client'

import styled from 'styled-components'

const Title = styled.h4`
  font-weight: bold;
  margin: 0;
`

export default Title
