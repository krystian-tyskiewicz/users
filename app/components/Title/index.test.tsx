import { render } from '@testing-library/react'
import Title from '.'

it('renders correctly', () => {
  const { container } = render(<Title>Test</Title>)
  expect(container).toMatchSnapshot()
})
