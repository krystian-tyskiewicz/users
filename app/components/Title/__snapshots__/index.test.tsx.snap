// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h4
    class="sc-bdvumH dNPaQk"
  >
    Test
  </h4>
</div>
`;
